static/docs/command-reference/update.md

```diff
@@ -39,6 +39,12 @@ to learn how to "update" fixed-revision imports.
 
 ## Options
 
+- `--rev` - specific
+  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+  (such as a branch name, a tag, or a commit hash) of the DVC repository to
+  update the data from. Repository's revision formed by the first import is used
+  by default when this option is not specified.
```

Comment:
> Repository's revision formed by the first import

I don't understand what this means :( Can you explain? This probably also depends on whether `--rev` was used with `dvc import` too.

Reply:
(Similar to Ivan's https://github.com/iterative/dvc.org/pull/890#pullrequestreview-337784630 but not exactly the same)

Reply:
updated by 82c8b5c, can you please re-check?

Reply:
updated by 82c8b5c, can you please re-check?

Reply:
Thanks, continued in https://github.com/iterative/dvc.org/pull/890#discussion_r363064106. Resolving this one.